tests/configs/k8s_with_node_selectors/kubetools.yml

```diff
@@ -0,0 +1,44 @@
+name: generic-app-with-node-selectors
+
+containerContexts:
+  generic-context:
+    image: generic-image
+    command: [generic-command]
+
+upgrades:
+  - name: Upgrade the database
+    containerContext: generic-context
+    nodeSelector:
+      nodeLabel: generic-label-value
+      nodeLabelName: generic-node-name
```

Comment:
These need to be more relevant, and less  confusing.

These tests are also here to document the capabilities of `kubetools` and, without going through k8s documentation, this makes it look like here we're defining 1 node selection label  with key `generic-node-name` and value `generic-node-value`...

Reply:
[2885b4c](https://github.com/EDITD/kubetools/pull/161/commits/2885b4c368dd4e58c2b4de1416f8d0dd3166cec2)